rentpath/ops
# Merge missing code from tag 'v1.0.0.pre'

[Bug](https://rentpath.leankit.com/card/536018216)

- refactor revision_spec.rb to represent latest changes
- removed unused files
- version bump

### How to test locally
* Copy files from `examples/sample_deploys/` to a root of another project.  
* Update Gemfile of another project, change to`gem 'ops', path: '../ops'`
* start a server and check `ops/version`
